Troubleshooting: add an article about gray boxes in grids (#5131)
DevExpress/devextreme-documentation: concepts/80 Troubleshooting/05 DataGrid and TreeList Issues/10 Gray Boxes Appear After Data Loading/15 Ensure Component Size Matches Container Size.md

> The component displays the placeholders based on its [height](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#height) property and the number of loaded rows. If the **height** property is not equal to the component's container size, the component does not load enough rows to fit the space. The remaining space is filled with gray boxes.

Reviewer: > The component displays **the** placeholders

Should we interpret this article case as definite? We haven't declared these "placeholders" earlier on in this document to refer to them as "definite".

> the component does not load enough rows to fit the space

This phrase is difficult to grasp. Especially the "fit" part. Looks like an incorrectly chosen word.

```suggestion
The component displays placeholders based on its [height](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#height) property and the number of loaded rows. If the **height** property value does not match the component's container size, the component does not load enough rows to fill the available area. The remaining area is filled with gray boxes.
```

> If none of the component's parent elements have a fixed height, the component cannot calculate its size correctly. If the height of DataGrid or TreeList's parent is relative, make sure that the parent's ancestor has a fixed height. If there are multiple ancestors with relative heights, at least one should have a fixed height.

Reviewer: Please avoid "there is/there are". You can always come up with a better solution.

```suggestion
If no parent element of the DataGrid or TreeList component has a fixed height, the component cannot calculate its size correctly. If the height of component's parent is relative, make sure that the parent's ancestor has a fixed height. If multiple ancestors have relative heights, at least one of them should have a fixed height.
```

> If you wish to fill the entire vertical space and therefore do not use fixed heights, set all parents (including `<html>`) heights to `100%`.

Reviewer: > set all parent**s** (including `<html>`) height**s** to `100%`.

An agreement of numbers issue. Should be: set all parent heights. Yes, it means "all heights of all parents".

To preserve your clarification enclosed in brackets, I had to rephrase all this.

```suggestion
If you wish to fill the entire container area by height and, therefore, do not use fixed heights, set the height of all parents (including `<html>`) to `100%`.
```

Please be more attentive and precise in your wordings.
